src/pages/Explorer/api.js

```diff
@@ -229,3 +230,24 @@ export const EntityQuery = {
       }
   }`,
 }
+
+function getStoreActivityMutation(entityType, entityId, interactionType) {
+  let normalizedEntityType = entityType
+
+  if (EntityType[entityType]) {
+    normalizedEntityType = EntityType[entityType].key
+  }
+
+  return `
+    mutation {
+        storeUserEntityInteraction(
+            entityType: ${normalizedEntityType}
+            entityId: ${entityId}
+            interactionType: ${interactionType}
+        )
+    } 
+  `
+}
+
+export const storeActivity = (entityType, entityId, interactionType) =>
```

Comment:
api functions' naming scheme: `[query|mutate][action]`
```suggestion
export const mutateStoreUserActivity = (entityType, entityId, interactionType) =>
```